specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json

```diff
@@ -8684,7 +8684,7 @@
           "items": {
             "$ref": "#/definitions/SubResource"
           },
-          "description": "Specifies an array of references to backend address pools of application gateways. A scale set can reference backend address pools of multiple application gateways. Multiple scale sets cannot use the same application gateway."
+          "description": "Specifies an array of references to backend address pools of application gateways. A scale set can reference backend address pools of multiple application gateways. Multiple scale sets can use the same application gateway."
```

Comment:
Multiple scale sets still cannot use the loadBalancerBackendAddressPools and loadBalancerInboundNatPools, right?

Reply:
Can you also update compute.json in the previous versions?  (2018-10-01/compute.json, etc)

Reply:
I didn't test these specifically, this change is only related to application gateways (I tested on AppGWv2).